Project.toml

```diff
@@ -6,6 +6,8 @@ version = "0.1.0"
 [deps]
 CLIMAParameters = "6eacf6c3-8458-43b9-ae03-caf5306d3d53"
 ClimaCore = "d414da3d-4745-48bb-8d80-42e94e092884"
+DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
```

Comment:
these should just be in the test environment. So to run your tests, use julia --project=test

Reply:
removed these, and started using julia --project=test